### DISCUSSION TITLE
Working with other music services?

### DISCUSSION THREAD

**Discussion by seanism** [vincentneo/LosslessSwitcher#52] [Ideas]
It would be great for this to work with other streaming / music services. Currently it does seem to change based on the quality in Tidal.

**Comment by Justinwang677** [comment]
> It would be great for this to work with other streaming / music services. Currently it does seem to change based on the quality in Tidal.

Adding QuickTime would be great too

**Comment by pato-gonzalez** [comment]
Does it work with other apps such as Tidal or is there any alternative?

**Reply by MasterRahool** [reply]
> Does it work with other apps such as Tidal or is there any alternative?

Tidal has exclusive mode built in...

**Reply by pato-gonzalez** [reply]
Okay so this wouldn't be necessary with Tidal? Tidal already changes MIDI output sample rate depending on the song's quality?

**Reply by MasterRahool** [reply]
> Okay so this wouldn't be necessary with Tidal? Tidal already changes MIDI output sample rate depending on the song's quality?

correct, this app is for Apple Music since it lacks the feature; for Tidal on desktop click the speaker symbol next to the volume control, then 'more settings' to toggle it on

<img width="637" alt="Screenshot 2024-06-24 at 10 19 43 AM" src="https://github.com/vincentneo/LosslessSwitcher/assets/104704170/83a62ddc-723c-438c-86db-e53fdcb70e3c">
<img width="545" alt="Screenshot 2024-06-24 at 10 19 38 AM" src="https://github.com/vincentneo/LosslessSwitcher/assets/104704170/a0e61339-3553-48c0-8836-d5f1fbff3b2a">

**Comment by codyghind** [comment]
Any chance the app gets Spotify support, now that Spotify provides Lossless FLAC quality?
